### DISCUSSION TITLE
Launch a non steam game directly without the need to launch a launcher first

### DISCUSSION THREAD

**Discussion by Zitagcc** [moraroy/NonSteamLaunchers-On-Steam-Deck#244] [Q&A]
Hi everyone, 

Is there a way to launch a non steam game without launching a launcher first. I didn't now how it is happened but I can launch Ghost recon directly and I would like to know how to do the same for my other non steam games.

**Comment by moraroy** [comment]
@Zitagcc that isn't the point of nsl. Nsl is meant to be launched with launcher on purpose, if you would like to do that without the launcher you can try heroic or lutris. Using the launcher is what allows multi-player, cloud saves, local saves, achievements, etc. Just like a windows pc.

**Comment by moraroy** [comment]
@Zitagcc I'm a little confused....are you saying you are confused about the shortcut that's in your library? If so, NSL creates that for you. It is using the launcher in the background. It's part of the NSL Game Scanner.

**Reply by Zitagcc** [reply]
Non, I wish to have shortcuts of my non steam games and I don't know how to do it. I don't want to open epic games launcher first in order to launch a game.

**Reply by moraroy** [reply]
@Zitagcc open nsl and hit stop game scanner and it will ask to restart, then download your game from your launcher. Then restart steam it will be there.

**Reply by Zitagcc** [reply]
I see now. Thank you so much for your help.
